pkg/app/request/metric_attributes.go

```diff
@@ -76,14 +76,14 @@ func ServerNamespaceMetric(val string) attribute.KeyValue {
 	return attribute.Key(attr.ServerNamespace).String(val)
 }
 
-func ConnectionTypeMetric(val string) attribute.KeyValue {
```

Comment:
has this been wrongfully deleted?

Reply:
i saw it wasn't used anywhere so it suggested to delete it
y-ebpf-instrumentation%20ConnectionTypeMetric&type=code